packages/common/src/store/ui/search-users-modal/sagas.ts

```diff
@@ -26,7 +26,9 @@ function* doSearchUsers(action: ReturnType<typeof searchUsers>) {
     })
     const users = Object.values(res.users) as User[]
     yield* call(processAndCacheUsers, users)
-    yield* put(searchUsersSucceeded({ userIds: users.map((u) => u.user_id) }))
+    yield* put(
+      searchUsersSucceeded({ userIds: users.map((u) => u.user_id), limit })
```

Comment:
I'm guessing the control paginates results as you scroll?

Reply:
Yup `onEndReached={handleLoadMore}` prop of flatlist and the action concats the results to the previous results.